tests/test_regressor/test_residuals.py

```diff
@@ -369,4 +372,4 @@ def test_alpha_param(self, mock_sca):
         # Test that alpha was passed to internal matplotlib scatterplot
         _, scatter_kwargs = visualizer.ax.scatter.call_args
         assert "alpha" in scatter_kwargs
-        assert scatter_kwargs["alpha"] == 0.3
+        assert scatter_kwargs["alpha"] == 0.75
```

Comment:
Awesome, this shows a good understanding that alpha is 0.3 after `visualizer.fit` then changes to 0.75 after `visualizer.score`

Reply:
Thanks!